homeassistant/components/aprilaire/entity.py

```diff
@@ -0,0 +1,83 @@
+"""Base functionality for Aprilaire entities."""
+
+from __future__ import annotations
+
+from pyaprilaire.const import Attribute
+
+from homeassistant.helpers.entity import DeviceInfo, Entity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+from homeassistant.util import slugify
+
+from .coordinator import AprilaireCoordinator
+
+
+class BaseAprilaireEntity(CoordinatorEntity, Entity):
+    """Base for Aprilaire entities."""
+
+    _attr_available = False
+    _attr_has_entity_name = True
```

Comment:
Oh nvm you already have this. In that case, please change the thermostat name to a translation key.

Reply:
Thanks, updated